combat-trainer.lic

```diff
@@ -357,7 +357,7 @@ class LootProcess
     end
     bput("get #{@gem_pouch_adjective} pouch from my #{@spare_gem_pouch_container}", 'You get a')
     bput('wear my pouch', 'You attach')
-    bput("stow #{item}", 'You pick')
+    bput('stow gem', 'You pick up', 'You get', 'You need a free hand', 'You just can\'t', 'push you over the item limit', 'You stop as you realize the .* is not yours', 'Stow what', 'already in your inventory')
```

Comment:
Should we check if `item` is in `@gem_nouns` like on line 339?

Reply:
The check was done a few lines earlier and we are no longer storing `item` in this case. If we check if item doesn't exist in the room, it will be the same result if the bput fails to `stow gem`.

Reply:
With stow gem we can avoid some inventory conflict though

Reply:
So we could handle the failure.. I see :)

Reply:
Does `stow gem` work for things like waermodi stones?

Reply:
I assume it does because the default gem loot option is `stow gem`. Its only during the pouch swap that we use the `item` variable.